aries-site/src/examples/components/layouts/FooterExample.js

```diff
@@ -8,11 +8,6 @@ export const FooterExample = () => {
     <Footer 
       background="background-front"
       direction={size !== 'small' ? 'row' : 'column-reverse'}
-      pad={{
-        vertical: size !== 'small' ? 'small' : 'large',
-        // Match horizontal padding of aries-core Nav
-        horizontal: size !== 'small' ? 'medium' : 'large',
-      }}
       align={size !== 'small' ? 'center' : undefined}
```

Comment:
Trying to comment something farther down this file, but the styling of the Footer Links doesn't match the most recent Footer styles (it's still using the bolded on hover).

Reply:
oh yes I changed that in the footer PR I just wasn't up to date with master should be good now!